tests/ui/apps/PopupTabLayout/NonAutoFocus/PopupTabLayoutNonAutoFocus-View.js

```diff
@@ -0,0 +1,95 @@
+import {useState} from 'react';
+import Group from '@enact/ui/Group';
+import ilib from '@enact/i18n';
+import {is} from '@enact/core/keymap';
+import spotlight from '@enact/spotlight';
+
+import {Header} from '../../../../../Panels';
+import Item from '../../../../../Item';
```

Comment:
Please change the import order
import Item from '../../../../../Item';
import {Header} from '../../../../../Panels';

Reply:
Suggested Changes.